examples/gnrc_networking_subnets/README.md

```diff
@@ -0,0 +1,78 @@
+# Auto-configuration for nested subnets on a (simple) tree topology
+
+This example demonstrates IPv6 subnet auto-configuration for networks on a
+tree topology.
+
+This allows to connect multiple links with individual subnets and route
+between them.
+Each link can have an arbitrary number of hosts, but there can be only
+a single router on each link.
+Routers can have multiple interfaces to connect different downlinks.
+
+![](../../doc/doxygen/src/gnrc_ipv6_auto_subnets_simple.svg)
+
+## Setup on native
+
+To simulate such a network on `native` a `setup_taps.sh` script is provided that
+will create TAP interfaces and bridges.
+Each bridge will be it's own subnet and `native` instances will act as routers
+to route between them.
+The setup script will also start `radvd` to advertise a large prefix that allows
+for sub-division by RIOT.
+
+The interfaces that should be created are specified in the `tapology.txt` file.
+
+Each router will have (at least) two interfaces that should be on different
+bridges.
+
+To start the first router, run
+
+    make term PORT="tap_a0 tap_b1"
+
+This will route between the `tap_a0` interface on `br0` and the `tap_b1`
+interface on `br1`.
+Start more `native` instances to simulate a cascading tree network:
+
+    make term PORT="tap_b0 tap_c1"
+    make term PORT="tap_c0 tap_d1"
+    …
+
+It is also possible to connect non-routing leaf nodes with a single interface:
+
+    make term LEAF=1 PORT=tap_b2
```

Comment:
I get

```
Terminal program /home/mlenders/Repositories/RIOT-OS/RIOT/examples/gnrc_networking_subnets/bin/native/gnrc_networking-subnets_leaf.elf is required but not found in PATH.  Aborting.
```

with that.

Reply:
Ok, will replace that with

    make all term LEAF=1 PORT=tap_b2